File: SpringBootJpa/src/test/java/com/example/java/TimeTests.java
package com.example.java;

import com.example.java.simple.domain.TimeData;
import com.example.java.simple.repository.TimeDataRepository;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

import java.time.LocalDateTime;

@SpringBootTest
class TimeTests {

    @Autowired TimeDataRepository timeDataRepository;

    @Test
    void test_save() {
        timeDataRepository.save(new TimeData(LocalDateTime.now()));
    }

}

File: SpringBootJpa/src/test/kotlin/com/example/kotlin/TimeTests.kt
package com.example.kotlin

import com.example.kotlin.simple.domain.TimeData
import com.example.kotlin.simple.repository.TimeDataRepository
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import java.time.LocalDateTime

@SpringBootTest
class TimeTests {

    @Autowired
    lateinit var timeDataRepository: TimeDataRepository

    @Test
    fun test_save() {
        timeDataRepository.save(TimeData(date = LocalDateTime.now()))
    }

}
